hack/vendor.sh

```diff
@@ -7,18 +7,17 @@
 set -e
 
 SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-"${SCRIPTDIR}"/go-mod-prepare.sh
 
 export GO111MODULE=on
```

Comment:
Having to set `GO111MODULE=on` everywhere with-go-mod.sh is used is tedious, and I can't imagine a situation where one would need to use the wrapper script with any other `GO111MODULE` value. Move this line into with-go-mod.sh?

Reply:
Good point!